SU-SWS/stanford_profile
# D8CORE-1697: Getting buttons to stack

# READY FOR REVIEW

# Summary
- When you put buttons sequentially on the page, they have a bunch of space in between. This is an approach to group the buttons. The trick is to put them inside the same `<p>`.

# Needed By (Date)
Before the next code freeze

# Urgency
Nice to have

# Steps to Test
1. Review code and verify this is an acceptable approach.
1. Pull this branch
1. Rebuild cache
2. Create a page with a WYSIWYG
2. Create at least 3 links with button styling each on separate lines
2. Create at least 3 links on a single line, and add button styling.
2. Save.
2. Verify that there is a different amount of spacing between the first set of buttons and the second.

![button_stacking](https://user-images.githubusercontent.com/284440/78839649-bfa84300-79ad-11ea-94f6-660185da4e6e.png)


# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1697
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
